kms/apiv1/requests.go

```diff
@@ -19,6 +19,35 @@ const (
 	HSM
 )
 
+// PINPolicy represents PIN requirements when signing or decrypting with an
+// asymmetric key in a given slot. PINPolicy is used by the YubiKey KMS.
+type PINPolicy int
+
+// PIN policies supported by this package. The values must match the ones in
+// github.com/go-piv/piv-go/piv.
```

Comment:
If they have to match, they could be aliased to the types in `go-piv`?

Reply:
@hslatman I don't want to depend on go-piv and cgo in this package.